### DISCUSSION TITLE
Danger Zone (Suspend your installation and Uninstall "Shopify")

### DISCUSSION THREAD

**Discussion by rajkumargwl** [community/community#61972] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Feature Area

Issues

### Body

Hello,

I want to know some clarification regarding the Danger zone is this related to do not updating anything here or it indicates that we need to remove/suspend those mentioned components? Please refer to the below screenshot for the same.
![image](https://github.com/community/community/assets/74894132/e3fb69a8-5bae-4755-ab09-b590440ab2df)

Thanks

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by nautics889** [comment]
Hello, @rajkumargwl 
The "Danger zone" sections contains management functional which represents either irreversible actions or global visibility/access settings.

On your screenshot, there are two options:
1. Suspend the app (block access for "Shopify" app, you can resume this later)
2. Uninstall app (uninstalling the app means you will lost the settings which you have and you would need to re-install the app again and configure it if you decided to re-setup it's again)

>it indicates that we need to remove/suspend those mentioned components

That definetely **does not mean** that you need to do any of those actions. You make decision yourself when it's time to suspend/uninstall the application (Shopify app in your case).

**Reply by chriswotp** [reply]
So what would happen if one DID NOT follow The request/suggestion?

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @rajkumargwl ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Comment by mehulrajdev** [comment]
Suspend the app (block access for the "Shopify" app, you can resume this later)
Uninstall app (uninstalling the app means you will lose the settings which you have and you would need to re-install the app again and configure it if you decided to re-setup it again)
